### PR TITLE
fix: problems encountered while building the effective model

### PR DESCRIPTION
**Pull request by dawidmalina**
fix: https://github.com/AzBuilder/terrakube/issues/1417

### CODE REVIEW

File: registry/pom.xml [hunk 1]
```diff
@@ -93,8 +89,8 @@
 			<version>${terrakube-client-starter.version}</version>
 			<exclusions>
 				<exclusion>
-					<groupId>com.squareup.okio:okio-jvm</groupId>
-					<artifactId>3.0.0 </artifactId>
+					<groupId>com.squareup.okio</groupId>
```
Comment (dawidmalina) [inline registry/pom.xml:92]: Looks like it was not working initially and when I fixed that tests start to fail. Removing that exclusion will fix the build. @alfespa17 what will be your suggestion here?

Comment (alfespa17) [reply registry/pom.xml:92]: I am not sure, this is why I never fixed the warning messages

Comment (dawidmalina) [reply registry/pom.xml:92]: I just double checked and it was not working. You can test it by doing that:
- check this branch (including proper exclusion):
```
cd registry
mvn help:effective-pom
```
- check main branch:
```
cd registry
mvn help:effective-pom
```

You will see that in the main branch effective pom does not contains that exclusion. In other words we should remove it completely.

Comment (dawidmalina) [reply registry/pom.xml]: Re-based with latest changes related to sonar build.

**Pull request merged**